config_db.go

```diff
@@ -27,6 +27,7 @@ var (
 	awsInstanceRoleKey     = bsonutil.MustHaveTag(Settings{}, "AWSInstanceRole")
 	cedarKey               = bsonutil.MustHaveTag(Settings{}, "Cedar")
 	hostJasperKey          = bsonutil.MustHaveTag(Settings{}, "HostJasper")
+	devProdOwnedBucketsKey = bsonutil.MustHaveTag(Settings{}, "DevProdOwnedBuckets")
```

Comment:
do we need to add this field [here](https://github.com/evergreen-ci/evergreen/blob/64270c1489e865190b90335cad43043d260ea999/config.go#L126-L155)?

Reply:
Moved to bucket config and I added it to the set method there!